### PR TITLE
Feat/add way to read a raw string

### PR DESCRIPTION
**Pull request by vinibispo**

### CODE REVIEW

File: lua/glow.lua [hunk 1]
```diff
@@ -220,6 +220,61 @@ local function open_window(path)
...
+  api.nvim_buf_set_keymap(
+    buf,
+    "n",
+    "<Esc>",
+    ":lua require('glow').close_window()<cr>",
+    { noremap = true, silent = true }
+  )
+
+  local use_pager = glow_use_pager and "-p" or ""
+  vim.fn.termopen(string.format("print '%s' | %s %s -s %s %s", str, glow_path, use_pager, glow_style, "-"))
```
Comment (ellisonleao) [inline lua/glow.lua:263]: what is print command ? for unix terminals we usually consider echo or printf

File: lua/glow.lua [hunk 2]
```diff
@@ -220,6 +220,61 @@ local function open_window(path)
...
+end
+
+function M.glow_string(string)
+  local current_win = vim.fn.win_getid()
```
Comment (ellisonleao) [inline lua/glow.lua:267]: this is a repeated logic from the glow function. How about trying to use the same for both commands?

File: lua/glow.lua (hunk 1 above)
Comment (vinibispo) [reply lua/glow.lua:263]: Ok, I'll change it

File: lua/glow.lua (hunk 2 above)
Comment (vinibispo) [reply lua/glow.lua:267]: Sure, I will do

**Pull request closed without merging**